packages/playground/blueprints/src/lib/steps/apply-wordpress-patches/wp-content/mu-plugins/5-log-sql-queries.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+function findAutoIncrementColumns() {
+    $pdo = new PDO('sqlite://wordpress/wp-content/database/.ht.sqlite');
+    $columns = [];
+
+    $stmt = $pdo->query("SELECT name FROM sqlite_master WHERE type='table' AND name NOT LIKE 'sqlite_%'");
+    $tables = $stmt->fetchAll(PDO::FETCH_COLUMN);
+    foreach ($tables as $table) {
+        $tableInfo = getTableInfo($pdo, $table);
+        $primaryKeyAutoIncrement = $tableInfo['primaryKeyAutoIncrement'];
+
+        if (!$primaryKeyAutoIncrement) {
+            continue;
+        }
+        $columns[$table] = $primaryKeyAutoIncrement;
+    }
+    unset($pdo);
+
+    return $columns;
```

Comment:
oh, lovely! I was thinking of rewriting the initial `CREATE` queries but this is much better, having sqlite update the primary auto-increment keys programmatically.

Reply:
❤️ 

Also, I went ahead and simplified this code a lot